src/servers/ZoneServer2016/entities/basefullcharacter.ts

```diff
@@ -399,6 +396,7 @@ export class BaseFullCharacter extends BaseLightweightCharacter {
       }
 
       Object.values(this._containers).forEach((c) => {
+        if (item.stackCount <= 0) return;
```

Comment:
What is the point of this? item.isValid already checks for this above.

Reply:
I was adding creating items with count 0 for each ewuipped container,

Reply:
ok